pkgs/applications/misc/candle/default.nix

```diff
@@ -24,9 +26,10 @@ mkDerivation rec {
   buildInputs = [ qtbase qtserialport ];
 
   meta = with lib; {
-    description = "GRBL controller application with G-Code visualizer written in Qt";
+    description =
+      "GRBL controller application with G-Code visualizer written in Qt";
```

Comment:
```suggestion
    description = "GRBL controller application with G-Code visualizer written in Qt";
```

Reply:
@SuperSandro2000 is there an official style guide for nixpkgs patches?  I believe that my editor used `nixpkgs-fmt` to reformat that file. I assumed that it was an approved formatter?

Reply:
nixpkgs-fmt does not do such things. nixfmt is doing stupidly short lines.